src/pylexibank/commands/check_languages.py

```diff
@@ -28,12 +28,19 @@ def check(ds, args):
         if gv != args.glottolog_version:
             warn('Dataset compiled against Glottolog {0}'.format(gv))
 
+    # no bookkeeping languages
     bookkeeping = set(
         l.id for l in args.glottolog.api.languoids() if l.lineage and l.lineage[0][1] == 'book1242')
     for l in cldf['LanguageTable']:
         if l['Glottocode'] and l['Glottocode'] in bookkeeping:
             warn('Language {0} mapped to Bookkeeping languoid {1}'.format(l['ID'], l['Glottocode']))
 
+    # lat and long should not be ALL empty
+    for col in ("Latitude", "Longitude"):
```

Comment:
It just occurs to me that referencing columns by name may be dangerous. For a dataset not created with `pylexibank`, such as Gereon's data, it is not safe to assume columns with the same propertyUrl will always have the same names.

(This also goes for my `''Glottocode'` reference above.)

Reply:
Hmm... ok, I guess we should think about what checks should be general (e.g. cldfbench) and what should be lexibank. I think the benefits of lexibank is consistency, so we should have a strict(er) set of rules for that, than something like cldfbench.

Reply:
i.e. I'm thinking most of this sort of thing could be lexibank checks https://github.com/lexibank/lexibank/blob/master/CONTRIBUTING.md

Reply:
...as well as more specific things like "was the CLDF data made from an old revision set of the github repository".

Reply:
But fixing the actual column names to be used is **not** something I would want to fix for lexibank, since this would encourage disregarding the fact that the data is CLDF. It would be similar to simply splitting TSV lines on `\t`, because most of the time one can get away with it ...

Reply:
I know that accessing columns via such a map is somewhat ugly, maybe `pycldf` should provide alternative ways to iterate rows of tables that already return mapped `dict`s.

Reply:
ok, right. 

What about using the CLDF metadata and terms.rdf to set properties on `Table` instances e.g. `cldf['LanguageTable'].glottocode` is whatever points to `http://cldf.clld.org/v1.0/terms.rdf#glottocode`. Then you'd easily know whatever fields are glottocode fields, so something like this would work:

```python
for row in cldf['LanguageTable']:
   glottocode = row[cldf['LanguageTable'].glottocode]
```

Not sure what you'd do if there's multiple glottocode terms, perhaps just the first?

Reply:
@SimonGreenhill with [this](https://github.com/cldf/pycldf/issues/86#issuecomment-579289901), we could mostly do what we wanted to do above.

Reply:
ok, let me move my suggestion over there and we can discuss there.